### ISSUE TITLE
Mighty Charge is Broken

### ISSUE THREAD

**Issue by WittleWolfie** [Vek17/TabletopTweaks-Core#16]
**Describe the bug**
The level 11 Cavalier feature Mighty Charge doesn't allow you to expand the variants.

**To Reproduce**
1. Level a Cavalier to 11
2. Click on the arrow above Mighty Charge

**Expected behavior**
Mighty Charge variants display for selection.

**Screenshots**
![image](https://user-images.githubusercontent.com/399167/202867420-3604e75d-0c4b-434b-a824-4f39e39c7db9.png)

**Additional context**
Stack Trace from log:

```
ArgumentNullException: Value cannot be null.
Paramater name: source
System.Linq.Enumerable.Select[TSource,TResult] () (at <351e49e2a5bf4fd6beabb458ce2255f3>:0)
TabletopTweaks.Core.NewUI.ActionBarPatches+ActionBarSlotVM_OnShowConvertRequest_Patch.Prefix () (at <e3a4160af4ad4c0b8422f6d4aae0f2ce>:0)
(wrapper dynamic-method) Kingmaker.UI.MVVM._VM.ActionBar.ActionBarSlotVM.Kingmaker.UI.MVVM._VM.ActionBar.ActionBarSlotVM.OnShowConvertRequest_Patch1(Kingmaker.UI.MVVM._VM.ActionBar.ActionBarSlotVM)
Owlcat.Runtime.UniRx.UniRxExtensionMethods+<>c__DisplayClass0_0.<Subscribe>b__0 () (at <6f43ac0656b645bc97d2b60f08481c9d>:0)
UniRx.Observer+Subscribe`1[T].OnNextInternal () (at <8278608da1374666822143d4d0b4e36e>:0)
UniRx.Observer+BaseObserver`1[T].OnNext () (at <8278608da1374666822143d4d0b4e36e>:0)
UniRx.Operators.FromEventObservable`1+FromEvent[TDelegate].OnNext () (at <8278608da1374666822143d4d0b4e36e>:0)
UnityEngine.Events.InvokableCall.Invoke () (at <25a85da7c6f04932b86e339dfd12957d>:0)
UnityEngine.Events.UnityEvent.Invoke () (at <25a85da7c6f04932b86e339dfd12957d>:0)
Owlcat.Runtime.UI.Controls.Button.OwlcatMultiButton.OnPointerClick () (at <04ff8e7c419d44869d9302a915971002>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute () (at <159744b43b644a6fbc060d1b0d4f50c2>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute[T] () (at <159744b43b644a6fbc060d1b0d4f50c2>:0)
UnityEngine.EventSystems.EventSystem:Update()
```

**Comment by Vek17** [comment]
Are you running the newest TTT-Core? This should have been fixed in tgat
afaik

**Comment by WittleWolfie** [comment]
This is on v0.5.5.

**Comment by Vek17** [comment]
Fixed om v0.5.6
